glotaran/analysis/test/test_scheme.py

```diff
@@ -23,13 +23,18 @@ def scheme(tmpdir_factory):
     dataset_path = path.join('dataset.nc')
     xr.DataArray([1, 2, 3]).to_dataset(name='data').to_netcdf(dataset_path)
 
+    extra_path = path.join('extradata.nc')
```

Comment:
Is this the only place where a nc datafile is generated and tested? This is a simple single data variable format of the nc format, not an entire full dataset. Is the latter also tested otherwise it may not be tested if open_dataarray vs open_dataset creates a problem down the line.

Reply:
I created https://github.com/glotaran/pyglotaran/issues/265 to address this